sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -486,4 +487,8 @@ public Message asMessage(BigInteger baseFee) {
     public byte[] encode(boolean accountSignature) {
         return EthereumTransactionEncoder.encodeAsRlpValues(this, accountSignature);
     }
```

Comment:
At the moment methods `encode(boolean accountSignature)` and `encode(boolean accountSignature, Writer writer)` have 2 independent implementations.
Could be better to make one backed by another. What do you think?

Reply:
I see the point,  I would  prefer to leave both of them since the rlp encoding with byte array is a little faster. Moreover they share some code for getting `EthereumTransaction` tx obj from `RlpList` and vice versa.

Reply:
ok, fine